src/components/Map/Map.tsx

```diff
@@ -1,6 +1,8 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './Map.css';
-import App from '../App/App';
+import QuizPage from '../QuizPage/QuizPage'
+const { v4: uuidv4 } = require('uuid')
```

Comment:
This is very helpful and solves the problem of using the country name as the key.